src/Compilers/VisualBasic/Portable/VBResources.resx

```diff
@@ -178,7 +178,10 @@
     <value>option '{0}' can be followed only by '+' or '-'</value>
   </data>
   <data name="ERR_InvalidSwitchValue" xml:space="preserve">
-    <value>the value '{0}' is invalid for option '{1}'</value>
+    <value>the value '{1}' is invalid for option '{0}'</value>
```

Comment:
Any particular reason you swapped this around? This is just going to confuse the translators.